README.md

```diff
@@ -9,11 +9,11 @@ To submit your code follow these simple steps:
 * Build your project including sources and send to Codota for indexing by running the following command in your project directory
     
     ```
-    mvn clean source:jar source:test-jar jar:test-jar package com.codota:codota-uploader:1.0.4:upload -Dcodota.token={your_token_here} -Dcodota.endpoint=https://upload.codota.com/artifacts -DskipAssembly -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip
+    mvn clean source:jar source:test-jar jar:test-jar package com.codota:codota-uploader:1.0.5:upload -Dcodota.token={your_token_here} -Dcodota.endpoint=https://upload.codota.com/artifacts -DskipAssembly -DskipTests -Dmaven.javadoc.skip=true -Dgpg.skip
     ```
     Also, you have the following optional parameters:
     ```
-    -Dcodota.repoName={your_repo_name} -Dcodota.lastPushed={utc_format_datetime}
+    -Dcodota.repoName={your_repo_name} -Dcodota.lastPushed={utc_format_datetime} -Dcodota.projectPrefix={prefix}
```

Comment:
Not related to your change, but I think repoName and lastPushed belong only in our internal version of this plugin for public code and don't here

Reply:
I'll remove it from README, but it's still supported.